src/commons/BrowserUtils.ts

```diff
@@ -784,23 +854,29 @@ export namespace BrowserUtils {
    * @param yOffset - Y (Pixels) offset to move to, relative to the top-left corner of the element. If not specified, the mouse will move to the middle of the element.
    */
   export function moveMouseCursorTo(
-    selector?: string,
+    selector: string,
     xOffset?: number,
     yOffset?: number
   ): void {
     Reporter.debug(
-      `Move mouse cursor to element: ${selector} - X: ${xOffset}, Y: ${yOffset}`
+      `Move mouse cursor to element: [${selector}] - X: [${xOffset}], Y: [${yOffset}]`
     );
-    browser.moveTo(selector, xOffset, yOffset);
+
+    if (xOffset === undefined || yOffset === undefined) {
```

Comment:
no case that you would like to pass only x or y offset?

Reply:
Currently wdio doesn't allow it (despite whats written in the documentation)  selector is a must.